src/ape/api/compiler.py

```diff
@@ -11,7 +11,7 @@
 class CompilerAPI:
     config: ConfigItem
 
-    @property
+    @property  # type: ignore
```

Comment:
Ugh pt. 2:

have to do this because mypy is not the smarted tool...

related to this change:
https://github.com/ApeWorX/ape/pull/302/files#r767063244

related bug:
https://github.com/python/mypy/issues/1362

Reply:
https://github.com/python/mypy/pull/11719/files  !!!

Reply:
Yay! We not longer have to do that!